### DISCUSSION TITLE
Can Windmill build complex workflow with high demand?

### DISCUSSION THREAD

**Discussion by nguyenthanhtien** [windmill-labs/windmill#3668] [Questions]
Our team wanna build complex workflow with windmill, and trigger with 100k CCU. It's possible?

**Comment by rubenfiszel** [comment]
It is possible but probably feasible much more easily using the dedicated workers for the high-throughput flows. This mode is EE only at the moment. It works by having the workers pre-load all the steps of a flow so that you never pay any cold start. It's as efficient as writing a dedicated micro-service for a flow.
